scripts/modelbench/inductor_test.sh

```diff
@@ -10,14 +11,40 @@ DT=${3:-float32}
 # static / dynamic
 SHAPE=${4:-static}
 LOG_DIR=${5:-inductor_log}
+DYNAMO_BENCH=${6:-1238ae3}
+# default / cpp
+WRAPPER=${7:-default}
+HF_TOKEN=${8:-hf_xx}
+
 mkdir -p $LOG_DIR
 
+export HUGGING_FACE_HUB_TOKEN=${HF_TOKEN}
+# https://github.com/pytorch/pytorch/issues/107200
+pip uninstall transformers -y && pip install transformers==4.30.2
+# collect sw info
+bash version_collect.sh ${LOG_DIR} ${DYNAMO_BENCH}
```

Comment:
I still suggest to move out the version collect part from `inductor_test.sh` to keep the clear purpose of this scripts, and also easy to use for others. Let's move out it and make them at same level